ouroboros-network/sim-tests-lib/Test/Ouroboros/Network/PeerSelection.hs

```diff
@@ -2074,21 +2156,27 @@ prop_governor_target_established_big_ledger_peers_below env =
                        --TODO: the environment does not yet cause this to happen
                        -- it requires synchronous failure in the establish action
                        Just (Set.singleton peer)
+                     TracePromoteWarmBigLedgerPeerFailed _ _ peer _ ->
+                       Just (Set.singleton peer)
                      --TODO: what about TraceDemoteWarmDone ?
                      -- these are also not immediate candidates
                      -- why does the property not fail for not tracking these?
                      TraceDemoteBigLedgerPeersAsynchronous status
                        | Set.null failures -> Nothing
                        | otherwise         -> Just failures
                        where
-                         failures = Map.keysSet (Map.filter (==PeerCold) . fmap fst $ status)
+                         failures = Map.keysSet (Map.filter (==PeerCooling) . fmap fst $ status)
```

Comment:
Should this be `<=PeerCooling`? Maybe not because `PeerCold` is only set as a consequence of `PeerCooling`, but I'd like to hear your opinion.

Reply:
It should be `==PeerCooling`, `PeerCold` peers might also exist but those transitions are of not particular interest for this test